### PR TITLE
Plat 4510 store cvv2

### PR DESCRIPTION
**Pull request by skynet67**

### CODE REVIEW

File: fitpay/src/main/java/com/fitpay/android/api/models/card/CreditCard.java [hunk 1]
```diff
@@ -170,7 +177,34 @@ public boolean canGetTransactions() {
...
+     * Update acceptTerms url
+     *
+     * <p>
+     * @see User#createCreditCard
+     * </p>
+     *
+     * @param acceptTermsUrl url
+     */
+    public void setAcceptTermsUrl(@NonNull String acceptTermsUrl) {
+        if (hasLink(ACCEPT_TERMS)) {
```
Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/api/models/card/CreditCard.java]: This should throw some type of exception, it means the card is not in a state to accept terms anymore

File: fitpay/src/main/java/com/fitpay/android/api/models/user/User.java [hunk 2]
```diff
@@ -234,6 +234,17 @@ public void onFailure(@ResultCode.Code int errorCode, String errorMessage) {
      * However, if the owner already has a default then it will not change.
      * To change the default, you should update the user to have a new "default_source".
      *
+     * <p>
+     * <b>Important note:</b>
```
Comment (ssteveli) [inline fitpay/src/main/java/com/fitpay/android/api/models/user/User.java]: So while this note is good, it needs to be more generic in that it states "use the hypermedia returned here", "getting the card again will not result in the proper hypermedia link".  Making references to the construct to the URL is something I want avoided in the SDK, we're changing this on the platform side.

File: fitpay/src/main/java/com/fitpay/android/api/models/card/CreditCard.java (hunk 1 above)
Comment (skynet67) [reply fitpay/src/main/java/com/fitpay/android/api/models/card/CreditCard.java]: done

File: fitpay/src/main/java/com/fitpay/android/api/models/user/User.java (hunk 2 above)
Comment (skynet67) [reply fitpay/src/main/java/com/fitpay/android/api/models/user/User.java]: done

**Pull request merged**